host-controller/src/main/java/org/jboss/as/host/controller/parsing/HostXml_10.java

```diff
@@ -423,9 +423,7 @@ private void parseHttpManagementInterfaceAttributes(XMLExtendedStreamReader read
                         break;
                     }
                     case ALLOWED_ORIGINS: {
-                        for (String origin : reader.getListAttributeValue(i)) {
-                            HttpManagementResourceDefinition.ALLOWED_ORIGINS.parseAndAddParameterElement(origin, addOp, reader);
-                        }
+                        HttpManagementResourceDefinition.ALLOWED_ORIGINS.getParser().parseAndSetParameter(HttpManagementResourceDefinition.ALLOWED_ORIGINS, value, addOp, reader);
```

Comment:
Just out of curiosity, should not we fix this also in the parsers for older model versions before 10?, I don't know if there could be possibilities of using older namespaces with newer server code. Just a doubt.

Reply:
me neither, I'm not aware such case. but if that's required, I can apply the same fix to old versions as well.

Reply:
Thanks @soul2zimate. If you want to send up a separate PR for older parsers that's fine but it's not required.  The steps to reproduce in the JIRA would result in the config being persisted, which would be with the latest namespace. So for an older namespace to be involved the "" value would have to be in the original xml, which is a corner case.